packages/core/src/components/Autocomplete/Autocomplete.scss

```diff
@@ -0,0 +1,78 @@
+// Form-specific variables can be found in _variables.forms
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Autocomplete
+
+@status beta
+
+Autocompletes allow users to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted), and receive one or more suggested matches in a list below the input.
+
+Style guide: components.autocomplete
+*/
+
+// autocomplete component
+.ds-c-autocomplete {
+  max-width: $input-max-width;
+  position: relative;
+}
+
+.ds-c-autocomplete__list {
+  background-color: $color-white;
+  box-sizing: border-box;
+  position: absolute;
+  width: 100%;
+  z-index: $z-dialog;
+}
+
+.ds-c-autocomplete__list-item {
+  color: $color-primary;
+  padding: $spacer-1 $spacer-2;
+}
+
+.ds-c-autocomplete__list-item--active {
+  background-color: $color-primary-alt-darkest;
+  color: $color-white;
+}
+
+.ds-c-autocomplete__list-item--message {
+  color: $color-gray-medium;
```

Comment:
Can use `$color-muted` here so it uses whatever the theme's muted color is. This is what we use for other things like form field hint text.

Reply:
Done. Switched to `$color-muted`.